sdk/js/src/aptos/client.ts

```diff
@@ -0,0 +1,52 @@
+import { AptosAccount, AptosClient, Types } from "aptos";
+
+export class AptosClientWrapper {
+  client: AptosClient;
+
+  constructor(nodeUrl: string) {
+    this.client = new AptosClient(nodeUrl);
+  }
+
+  executeEntryFunction = async (
+    sender: AptosAccount,
+    payload: Types.EntryFunctionPayload,
+    opts?: Partial<Types.SubmitTransactionRequest>
+  ) => {
```

Comment:
can you add return types explicitly? it makes the code a little easier to read outside of an IDE

Reply:
ahh yes I'll start doing this to accommodate my non-vscode brothers